src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1581,7 +1503,7 @@
     parameters:
         - name: --communities
           short-summary: |
-                Space separated list of BGP community values to filter on. (e.g.: 12076:5010)
```

Comment:
Keep the example value or add an example that shows how it is used. Otherwise the user may not know what to pull from the populator command to use this parameter.

Reply:
What is the acronym definition for BGP so that it can be included in the summary?

Reply:
border gateway protocol

Reply:
Border Gateway Routing

Reply:
Wouldn't that be BGR then? :smirk: